hardware/Layout - desing- guide/Layout-Desing-Guide.md

```diff
@@ -0,0 +1,464 @@
+
```

Comment:
1. The md file name should be "Layout-Design-Guide.md", right? 
2. The md file should be put under **"hardware"** directory, not under another sub-diretory. 
3. The image files should be put under **"hardware/files/HW-Layout-Design-Guide/"**  folder and **"files/HW-Layout-Design-Guide/"**

Reply:
Forgot the mention, please also update the hardware-bootcamp.md and _Sidebar.md for your article displayed at the page.

Reply:
Done